src/Service/FileUploader.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace App\Service;
+
+use Symfony\Component\HttpFoundation\File\Exception\FileException;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class FileUploader
+{
+    private $targetDir;
+
+    public function __construct($targetDir)
+    {
+        $this->targetDir = $targetDir;
+    }
+
+    public function upload(string $path, UploadedFile $file): string
+    {
+        $fileName = $this->generateUniqueFileName() . '.' . $file->guessExtension();
```

Comment:
`{$this->generateUniqueFileName()}.{$file->guessExtension()}`

Reply:
Resolve